Automattic/wp-calypso
# Update Jetpack biennial upsell

Related to https://github.com/Automattic/jetpack-marketing/issues/497

This PR will also resolve maintenance issue https://github.com/Automattic/jetpack-marketing/issues/479

## Proposed Changes

We're updating the biennial upsell of Jetpack products in Checkout to be more straightforward in presenting the benefits to customers about subscribing for two years directly.

Before:
![CleanShot 2024-01-24 at 16 12 18@2x](https://github.com/Automattic/wp-calypso/assets/8419292/b7bca18f-0d9e-4d9d-845e-c633b0f10976)

After:
![CleanShot 2024-01-24 at 16 11 08@2x](https://github.com/Automattic/wp-calypso/assets/8419292/0409a331-24e0-4b62-baf3-e0318c5052af)



## Testing Instructions

* Use Calypso Live or checkout the PR locally
* Go to /checkout/jetpack/jetpack_complete
* Play with different use cases: incognito, with a12n account and credits applied, with tax region (e.g. Spain and ZIP 28004), with coupon (https://github.com/Automattic/jetpack-marketing/issues/497#issuecomment-1908338859).
* Check also /checkout/jetpack/jetpack_social_advanced_yearly that is a special case of free trial instead of intro offer. it should look like below:
![CleanShot 2024-01-24 at 16 16 40@2x](https://github.com/Automattic/wp-calypso/assets/8419292/3c89b534-6ce8-4908-af26-a2fa2bd4a070)
